docs/source/guides/models.rst

```diff
@@ -0,0 +1,87 @@
+Creating Models
+~~~~~~~~~~~~~~~
+
+.. include:: ../imports.rst
+
+Most neural network architectures contain multiple links.
+For example, a multi-layer perceptron consists of multiple linear layers.
+We can write complex procedures with parameters by combining multiple links like this:
+
+.. doctest::
+
+   >>> l1 = L.Linear(4, 3)
```

Comment:
I am not too comfortable with doctest but does this code pass (looks like L is not defined)?

Reply:
I believe so, because of the `.. include:: ../imports.rst` above.